data/campaigns/Eastern_Invasion/scenarios/14_The_Drowned_Plains.cfg

```diff
@@ -0,0 +1,2037 @@
+
+            {GENERIC_UNIT 6 ({TYPE}) {X} {Y}} {ANIMATE}
+            {MOVE_UNIT (x,y={X},{Y}) {TO_X} {TO_Y}}
+        [/then]
+    [/if]
+#enddef
+#define UNGATE X Y
+    {MODIFY_TERRAIN Ds {X} {Y}}
+    [sound]
+        name=net.wav
+    [/sound]
+    [delay]
+        time=200
+    [/delay]
+
+    [redraw]
+        clear_shroud,side=yes,1
+    [/redraw]
+#enddef
+    [event]
+        name=prestart
+
+        {SET_LABEL 16 24 _"Lord Alric’s Palace"  }
+        {SET_LABEL 17 5  _"Sir Efran’s Castle"   }
+        {SET_LABEL 7 50  _"Lord Gaelyc’s Citadel"}
+        {SET_LABEL 68 42 _"Sir Seoraery’s Keep"  }
+        {SET_LABEL 49 9  _"Clearwater Lake"      }
+
+        {PLACE_IMAGE "items/bones-elevated.png"     16 24} # lord alric
+        [item]
+            x,y,image=7,50,items/bones-elevated.png~FL() # lord gaelyc
+            visible_in_fog=no
+        [/item]
+
+        # the path we came from
+        [remove_shroud]
+            x,y,radius=73,23,4
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=69,25,4
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=65,28,4
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=70,18,4
+        [/remove_shroud]
+
+        # Owaec's memories. To make the map less mysterious
+        [remove_shroud] # a few reveals spread around the lake, to make it look more natural
+            x,y,radius=50,8,6
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=47,9,6
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=48,6,6
+        [/remove_shroud]
+        [remove_shroud]
+            x,y,radius=50,6,6
+        [/remove_shroud]
+
+        # holy amulet, so players are less likely to miss it
+        [remove_shroud]
+            x,y,radius=43,12,1
+        [/remove_shroud]
+
+        [objectives]
+            side=1
+            [objective]
+                description= _ "Defeat the spirits"
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Gweddry or Dacyn"
+                condition=lose
+            [/objective]
+        [/objectives]
+    [/event]
+    [event]
+        name=start
+
+        {RECALL_XY Dacyn 56 28}
+        [redraw]
+        [/redraw]
+
+        {RECALL_XY Owaec 55 28}
+        {RECALL_XY Addogin 57 27}
+        {RECALL_XY (Hahid al-Ali) 57 27}
+        {RECALL_XY Terraent 57 27}
+        {RECALL_XY Dolburras 57 28} # choose tiles that won't get in the way of the gate
+        {RECALL_XY Grug 57 28}
+        {RECALL_XY Gaennell 56 26}
+        {MODIFY_UNIT side=1 facing sw}
+        [redraw]
+            clear_shroud=yes
+            side=1
+        [/redraw]
+
+        [message]
+            speaker=Owaec
+            message= _ "At last we have reached the Horse Plains, the fair homeland of my people! Here we can finally reuinte with our comrades-in-arms and vanquish the undead once-and-for-all."
```

Comment:
I don't think it's necessary to hyphenate 'once and for all'.

Reply:
Not only is it unnecessary, it is wrong.

I also notice the correct pluralization of "comrade-in-arms" is used here, so that's nice. On the other hand, "reunite" is misspelled.

Reply:
The same phrase is used 'once and for all' elsewhere in the dialogue too.

How else would comrades-in-arms be plural?

Oops, I completely missed 'reuinte'.